tests/helpers/calc/test_math_array.py

```diff
@@ -126,7 +127,7 @@ def test_addition_with_other_types():
     with raises(ShapeError, match=match):
         A + (1 + 2j)
     with raises(TypeError, match="Cannot add/subtract a matrix with object of "
-                "<type 'list'>"):
+                "{list_type}".format(list_type=list)):
```

Comment:
Did you mean `type(list)`?

Reply:
No, because `type(list)` is `<type 'type'>` in Python 2 and `<class 'type'>` in Python 3. We want the type of the instance (name,y, `list`), not the type of the class/type.

Also works: `.format(type_list=type([])`, but it annoyed me to instantiate a list for no reason, plus I found `.format(type_tuple=type(tuple()))` to be ugly.

Reply:
:+1: